src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const gallery = document.querySelector('.gallery');
+  const mainImage = document.querySelector('#largeImg');
+
+  gallery.addEventListener('click', (e) => {
+    const thumbnail =
+      e.target.tagName === 'img'
+        ? e.target
+        : e.target.tagName === 'a' && e.target.querySelector('img');
```

Comment:
The condition `e.target.tagName === 'img'` and `e.target.tagName === 'a'` should use uppercase tag names, i.e., 'IMG' and 'A', because `tagName` returns the tag name in uppercase.